inference-engine/samples/classification_sample_async/main.cpp

```diff
@@ -8,25 +8,29 @@
  * @example classification_sample_async/main.cpp
  */
 
-#include <format_reader_ptr.h>
-#include <samples/classification_results.h>
 #include <sys/stat.h>
 
 #include <condition_variable>
 #include <fstream>
-#include <inference_engine.hpp>
 #include <map>
 #include <memory>
 #include <mutex>
-#include <samples/args_helper.hpp>
-#include <samples/common.hpp>
-#include <samples/slog.hpp>
 #include <string>
 #include <vector>
 
-#include "classification_sample_async.h"
+// clang-format off
+#include "inference_engine.hpp"
```

Comment:
Why do you have old headers in new sample?

Reply:
removed